mslib/msui/mscolab.py

```diff
@@ -260,6 +260,9 @@ def connect_handler(self):
                     self.loginButton.setEnabled(True)
             else:
                 show_popup(self, "Error", "Some unexpected error occurred. Please try again.")
+        except requests.exceptions.SSLError:
+            logging.debug("Certificate Verification Failed")
+            show_popup(self, "Error", "Certificate Verification Failed")
```

Comment:
good to move this before ConnectionError, https://github.com/Open-MSS/MSS/issues/673

Perhaps moving this ConnectionError to the buttom and a comment will have no impact for others.

Reply:
ConnectionError is now the last requests Exception ot be caught.